docs/ibexa_products/editions.md

```diff
@@ -8,7 +8,41 @@ page_type: landing_page
 Three [[= product_name =]] product editions are available to help you accelerate your digital transformation at the speed and cost that work best for you.
 
 [[= cards([
-"ibexa_products/headless",
-"ibexa_products/experience",
-"https://www.ibexa.co/products/ibexa-commerce"
-], columns=3) =]]
\ No newline at end of file
+"ibexa_products/ibexa_headless",
+"ibexa_products/ibexa_experience",
+"ibexa_products/ibexa_commerce"
+], columns=3) =]]
+
+## Feature comparison
+
+Compare all features available in [[= product_name_headless =]], [[= product_name_exp =]], and [[= product_name_com =]] to help you choose the right products for your needs:
```

Comment:
I really, really like this table 💪 

A couple of suggestions:
1) I'd consider adding:
Headless:
- Migrations
- Connect
- OAuth (client and server)

Experience:
- Activity log (Recent activity)

Commerce:
- Checkout
- Cart

2) "Publish later" is the same thing as "Content scheduler" - available in Headless. In Experience the feature is expanded with block scheduling and scheduler block (not sure if we want to include it)
3) I'm not sure about naming the workflow feature "Collaboriation workspace" - we're about to introduce a new feature called Collaboration that will conflict with it